lib/improver/tests/wind_downscaling/test_roughnesscorrection.py

```diff
@@ -113,28 +112,28 @@ def set_up_cube(num_time_points=1, num_grid_points=1, num_height_levels=7,
         try:
             data = np.array(data)
             cube.data = data.reshape(cube.data.shape)
-        except ValueError as ex:
-            if ex.message == "total size of new array must be unchanged":
+        except ValueError as error_x:
+            if error_x.message == "total size of new array must be unchanged":
                 msg = ("supplied data does not fit the cube."
                        "cube dimensions: {} vs. supplied data {}")
                 raise ValueError(msg.format(cube.shape, data.shape))
             else:
-                raise ValueError(ex)
+                raise ValueError(error_x)
 
     if name is not None:
         try:
             cube.standard_name = name
-        except ValueError as ex:
+        except ValueError as _error:
             msg = "error trying to set the supplied name as cube data name: "
-            raise ValueError(msg + ex.message)
-        except TypeError as ex:
+            raise ValueError(msg + error_x.message)
```

Comment:
Should this be _error rather than error_x?

Reply:
Yes

Reply:
Done